src/java.base/share/man/java.1

```diff
@@ -4392,7 +4391,7 @@ start the VM with the command line option:
 .RE
 .PP
 See \f[B]CodeHeap State Analytics (OpenJDK)\f[R]
-[https://bugs.openjdk.java.net/secure/attachment/75649/JVM_CodeHeap_StateAnalytics_V2.pdf]
+[https://bugs.openjdk.org/secure/attachment/75649/JVM_CodeHeap_StateAnalytics_V2.pdf]
```

Comment:
A reflection is that this does not seem like a good permanent URL to refer to in our specifications; but I'll leave that to someone else to fix.